app/models/user.rb

```diff
@@ -18,10 +18,8 @@ def full_name=(new_name)
     self.last_name = names.join(" ")
   end
 
-  def gravatar(size = 500)
-    "http://www.gravatar.com/avatar/#{ Digest::MD5.hexdigest(email.downcase) }?s=#{ size }&d=retro"
-  rescue
-    avatar_url.presence || random_avatar
+  def avatar(size: 512)
```

Comment:
Unused method argument - `size`. You can also write as `avatar(*)` if you want the method to accept any arguments but don't care about them.